src/App.js

```diff
@@ -1,28 +1,51 @@
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import { useState } from 'react';
-import './App.css';
```

Comment:
Did you delete these stylesheet import statements by mistake?

Reply:
Yes, great catch! I accidentally used Router V5 syntax and had to refactor import statements. it was mistakenly deleted at that time. Can this be fixed in a followup PR?

Reply:
Certainly. Just make sure to push the change before moving on to anything else.

Reply:
Will do thanks!